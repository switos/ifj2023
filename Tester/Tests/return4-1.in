func addNumbers(a: Int, b: Int) -> Int {
    return a + b
}

let result = addNumbers(a: 5)
