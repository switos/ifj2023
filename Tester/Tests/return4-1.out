Expected Output:

Expected Return Code: 4
